rfcs/react-components/components/simple-component-implementation.md

```diff
@@ -0,0 +1,125 @@
+
+It is also important to note that this option doesn't mean we will provide code snippets that you have to copy paste, but rather guide the user on what's the important part of the component and leave the rest to the user.
+
+#### Folder structure inside docs
+
+```
+\- Concepts
+  |- Introduction
+  |- Developer
+  |- Migration
+  |- Recipes
+    |- Recipe #1
+    |- Recipe #2
+\- Theme
+  |- Color
+  |- Motion and Size
+  |- Typography
+\- Components
+  |- ...
+\- Preview Components
+  |- ...
+```
+
+#### Pros
+
+- No need for a new package in our repo.
+- In the case of a component like persona where its design requires us to change the icon's size, we would provide a "best practices" section that could include a size table for reference.
+
+#### Cons
+
+- Allows for issues where the user-created component doesn't look like the design.
+- Could be overwhelming for the user to follow the recipe.
+
+### Option 2: We provide the component
+
+For this option, we would implement the component and provide it like any other component.
+
+#### Pros
+
+- We can ensure the design follows the design spec.
+- Makes it easier for the user.
+
+#### Cons
+
+- Another component package for a simple component.
+- More code to maintain.
+- More design figmas.
+
+### Option 3: Option 1 + Option 2 `@fluentui/react-recipe-hooks`
+
+We provide style hooks such as `useMediaObjectStyles()` or `useMediaObjectClassNames()`. The hook's options would contain what would be the props in a regular component and it would return its classnames so the user can apply them respectively. These hooks could be housed in a package such as `@fluentui/react-recipe-hooks`. A simple example could be:
+
+```jsx
+import { Text } from '@fluentui/react-text';
+import { Image } from '@fluentui/react-image';
+import { useMediaObjectClassNames } from '@fluentui/react-recipe-hooks';
+
+const MyMediaObject = () => {
+  const mediaObjectClassNames = useMediaObjectClassNames({ labelPosition: 'below' });
+
+  return (
+    <div className={mediaObjectClassNames.mediaObject}>
+      <Image
+        className={mediaObjectClassNames.media}
+        block
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/100x100.png"
+      />
+      <Text className={mediaObjectClassNames.text}>Lorem Ipsum</Text>
+      <Text className={mediaObjectClassNames.text}>Dolorem</Text>
+    </div>
+  );
+};
+```
+
+There will still be parts of the component that the user will have to implement, these hooks will not be a full implementation.
+
+#### Pros
+
+- We can add more design guidance.
+- While the user still has to do more work than option 2, it's way easier than option 1.
+- Simpler to maintain than option 2.
+- Can make option 1 easier to follow.
+
+#### Cons
+
+- More code to maintain.
+- Could bring more issues by being hard to follow.
```

Comment:
You mentioned accessibility issues for Option #1, but I think it's still an issue for this option as it provides only styling.